ts/features/appReviews/store/reducers/index.ts

```diff
@@ -0,0 +1,60 @@
+import { getType } from "typesafe-actions";
+import { PersistConfig, persistReducer } from "redux-persist";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { Action } from "../../../../store/actions/types";
+import {
+  appReviewNegativeFeedback,
+  appReviewPositiveFeedback,
+  clearFeedbackDatas,
+  TopicKeys
+} from "../actions";
+
+export type AppFeedbackState = {
+  positiveFeedbackDate: string;
+  negativeFeedbackDate:
+    | Record<TopicKeys, string | undefined>
+    | Record<string, never>;
+};
```

Comment:
What about using  `IsoDateFromString` from `io-ts-common`? What do you think?

Reply:
the string is at the initial state an empty string, it is not that wrong to change all the logic to properly handle the undefined instead of an empty string, but I think that would not bring that benefits in this case since we write a iso string date directly from the reducer.

What do you think?

Reply:
I’ll leave the final decision to you. Personally, I prefer using a strict type like `IsoDateFromString` as it enforces correctness and leads to more readable and maintainable code. That said, if handling all stuff at the reducer level already keeps things consistent and readable, I see your point. Let me know what you think!

Reply:
Got your point, `IsoDateFromString` is actually a `Date` object not a string, since we're persisting the value to get it through app sessions, maybe it would be better even for future possible migrations to keep it as a simple string instead of using a complex value as a Date, I would prefer to keep it as it is. Don't know if I'm missing some point at the moment, what do you think?

Reply:
You're right. I took the wrong type. What I mean is to have a type following this regex `UTC_ISO8601_FULL_REGEX` from `io-ts-common`. Because nothing tells me that inside 
- `positiveFeedbackDate`
- `negativeFeedbackDate`
I must expect a string following the `UTC_ISO8601_FULL_REGEX`.

Reply:
BTW. I just wanted to pint this out, but we can leave things the way they are. Maybe just a comment explaining the type'd be enough.